Scripts/BuildPhases/GenerateCredentials.sh

```diff
@@ -52,6 +52,12 @@ ensure_is_in_input_files_list $WORDPRESS_SECRETS_FILE
 JETPACK_SECRETS_FILE="${SECRETS_ROOT}/Jetpack-Secrets.swift"
 ensure_is_in_input_files_list $JETPACK_SECRETS_FILE
 
+# TODO: We'll need a dedicated one for Reader...
+# READER_SECRETS_FILE="${SECRETS_ROOT}/Reader-Secrets.swift"
+# ...using WordPress-Secrets.swift for now.
+READER_SECRETS_FILE="${SECRETS_ROOT}/WordPress-Secrets.swift"
+ensure_is_in_input_files_list $READER_SECRETS_FILE
```

Comment:
Just waiting for the appropriate entries to be created. See also p1744279985523659/1744233208.119539-slack-C08LE4U7FAM